projects/plugins/protect/src/js/components/product-offer/index.jsx

```diff
@@ -13,21 +17,26 @@ import useProtectData from '../../hooks/use-protect-data';
  * @param {Function} props.onAdd      - Callback for Call To Action button click
  * @returns {object}                    ConnectedProductOffer react component.
  */
-const ConnectedProductOffer = ( { onAdd, ...rest } ) => {
+const ConnectedProductOffer = ( { onAdd, onAddError, ...rest } ) => {
```

Comment:
missing doc for the new onAddError prop.

Again, should we document it here or down below in the propTypes? We are not following a standarad. Does Storybook read it from here? When I started developing components it didn't, that's why I started documenting on propTypes section

Reply:
I can add the doc in the propTypes component property